### DISCUSSION TITLE
Update Twitter(X) Logo in Social accounts

### DISCUSSION THREAD

**Discussion by parthdasawant** [community/community#68219] [Other Feature Feedback, Questions, & Ideas]
### Select Topic Area

Bug

### Body

Update old Twitter(X) Logo in Social accounts to new one.

**Comment by itsdotnet** [comment]
Its not changed yet.
Even in America Twitter Users are not agree with rebranding

**Reply by parthdasawant** [reply]
It's changed on [github about](https://github.com/about) page in footer

<img width="1702" alt="Screenshot 2023-09-26 at 12 49 25 PM" src="https://github.com/community/community/assets/80618762/19ff6cf1-4eee-4b64-b707-c4af4d605ccc">




But still they are using old one at same About page in Follow us on section :) 
<img width="1920" alt="Screenshot 2023-09-26 at 12 57 36 PM" src="https://github.com/community/community/assets/80618762/84a244cd-2d2a-41e8-940b-b69c8dc06d82">

**Reply by itsdotnet** [reply]
So that means @github Frontend devs are so lazy

**Comment by FIOpwK** [comment]
If it helps, here is my profile page with Larry in white. Twitter's logo is an asset. It is their bird. The brand name 'Twitter' advises to "Don't use outdated version of the logo."  See: https://tinyurl.com/twitter-trademark -- **[Page 4 , Point 4].**  

In short, changing the Twitter logo on the Github profile page from Larry to X Corp complies with the Developer Policy associated with the brand name "Twitter". It now means X.Corp. See: https://tinyurl.com/XCorpDevPolicy **[Section: 1.13] and [Section: 4A]**. 

If I have an opinion, it is that both logos are still valid. Contact trademarks@twitter.com for sound arguments because honestly, even Twitter still uses Larry on their Github profile: https://github.com/topics/twitter. 

![FIOpwK (Dajuan Mcdonald) Screenshot](https://github.com/community/community/assets/16410995/42b1ee78-e23f-4106-80d1-b18416d9e0fe)

**Reply by parthdasawant** [reply]
Thanks for sharing this!
